manifest.yaml

```diff
@@ -172,7 +172,7 @@ arguments:
         - number
   versions.grpcClients.nodejs:
     description: Nodejs version to use for gRPC clients
-    default: "16.19.0"
+    default: "18.17.1"
```

Comment:
I think we're stuck with this until Giraffe upgrades, technically. We could just upgrade it anyways

Reply:
Is giraffe bootstrapped? I can't recall the difference between Giraffe and Gazelle, and if Gazelle has replaced Giraffe yet.

Reply:
> Is giraffe bootstrapped?

No

> if Gazelle has replaced Giraffe yet

Not yet.